packages/css/src/logo/README.md

```diff
@@ -0,0 +1,35 @@
+# Logo
+
+Het logo van de gemeente bestaat uit een beeldmerk (3 Andreaskruisen) en een woordmerk (Gemeente Amsterdam).
+Het logo heeft een vaste kleur, grootte en positie en staat op al onze websites.
+
+## Specificaties
+
+Gebruik altijd het online logo.
+Het Nederlandse logo voor Nederlandstalige websites en anderstalige websites behalve Engels.
+Het Engelstalige logo gebruiken we voor Engelstalige websites.
+
+## Uitzondering
+
+Gebruik het submerklogo als de website eigendom is van 1 van de 4 submerken.
+Zij hebben een aparte status omdat er sprake is van een bijzondere vorm van dienstverlening die niet direct met de gemeente in verband wordt gebracht.
+
+De submerken zijn:
+
+- GGD Amsterdam
+- Stadsarchief
+- Stadsbank van Lening
+- VGA Verzekeringen
+
+## Eisen voor alle logo´s
+
+- Het logo wordt altijd getoond.
+- Het logo is linksboven uitgelijnd in de header.
+- Het logo verwijst naar de homepage.
+- Als de applicatie een formulier, applicatie of tool is zonder homepage, dan linkt het logo naar de pagina op Amsterdam.nl van waar naar het formulier, applicatie of tool wordt verwezen.
```

Comment:
This isn't necessarily amsterdam.nl, right?

```suggestion
- Als de applicatie een formulier, applicatie of tool is zonder homepage, dan linkt het logo naar de pagina van waar naar het formulier, applicatie of tool wordt verwezen.
```

Reply:
Sounds fair.